samples/CommunityToolkit.Maui.Sample/ViewModels/ViewControls/AvatarModel.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace CommunityToolkit.Maui.Sample.ViewModels.ViewControls;
+
+public class AvatarModel
+{
+	public Color? AvatarBackgroundColor { get; set; } = Colors.Black;
+	public FontAttributes? FontAttributes { get; set; }
+
+	public double? FontSize { get; set; } = Device.GetNamedSize(NamedSize.Large, typeof(Label));
+
+	public ImageSource? ImageSource { get; set; }
+
+	public string? Text { get; set; } = "?";
+
+	public Color? TextColor { get; set; } = Colors.White;
+
+	public Thickness? BorderWidth { get; set; } = 0;
```

Comment:
Please make this Property read-only and remove nullability 
```suggestion
	public Thickness BorderWidth { get; init; } = 0;
```